src/main/java/blackjack/domain/judge/Judge.java

```diff
@@ -0,0 +1,51 @@
+package blackjack.domain.judge;
+
+import blackjack.domain.participant.Participant;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Judge {
+
+    private static final int LIMIT_SCORE = 21;
+    private static final String PLAYER_WIN = "승";
+    private static final String PLAYER_LOSE = "패";
+    private static final String DEALER_WIN_OR_LOSE_FORMAT = "%d승 %d패";
+
+    public Map<String, String> getWinOrLose(final Participant dealer, final List<Participant> players) {
```

Comment:
여기서는 첫번재는 Dealer, 두번째는 Player라는게 보장되어야하기 때문에
아래와 같이 구현 타입을 받아주면 어떨까요?

```suggestion
    public Map<String, String> getWinOrLose(final Dealer dealer, final List<Player> players) {
```